### ISSUE TITLE
Animation when skeleton disappears and shows the actual content

### ISSUE THREAD

**Issue by CSolanaM** [CSolanaM/SkeletonUI#4]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a transition when skeleton finishes and shows the real content.

**Describe the solution you'd like**
A fade-out opacity animation when the skeleton finishes loading and a fade-in opacity animation on the real content running in parallel.

**Describe alternatives you've considered**
Supporting other kind of transitions.
